assets/js/page-enhancements.js

```diff
@@ -30,6 +30,24 @@
         selectElement: element,
       })
     })
+    $('#create-new-contact-form').submit(event => {
```

Comment:
I wouldn't normally comment to this degree, but this is such a nasty hack and the fact that we needed to do it in the first place left me feeling that we should document it for fear of someone inadvertently removing/changing this in the future (even though we do have the cypress tests introduced in this PR to protect us)